int8 n_moved_blocks
---
geometry_msgs/Pose[] poses
string[] blocks_id
int8 n_blocks
bool finished